cls/SourceControl/Git/Utils.cls

```diff
@@ -2062,5 +2063,74 @@ ClassMethod ResetSourceControlClass()
     do ##class(%Studio.SourceControl.Interface).SourceControlClassSet("")
 }
 
+ClassMethod BaselineExport(pCommitMessage = "", pPushToRemote = "") As %Status
+{
+    set sc = $$$OK
+    try {
+        write !, "Exporting classes..."
+        set sql = "SELECT ID FROM %Dictionary.ClassDefinition"
```

Comment:
I would recommend using [%Library.RoutineMgr:StudioOpenDialog](https://docs.intersystems.com/irislatest/csp/documatic/%25CSP.Documatic.cls?LIBRARY=%25SYS&CLASSNAME=%25Library.RoutineMgr#StudioOpenDialog) instead as it can filter out mapped items better.